### DISCUSSION TITLE
Set default value for custom script field?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#15531] [Q&A]
I’m thinking about building a custom script that would be used to quickly add new devices. Somewhat similar to the default Add Device page, but stripped down to only the fields we use. And it would include primary IP address, MAC, power, and Ethernet cable connections. 

I’m wondering if there is some way to set the default value for certain ObjectVar fields. For example, if I’m adding 20 devices at the same site/location, I’d like to pre-fill those 2 fields to the most recently used value every time the script is run from the GUI.

**Comment by alehaa** [comment]
Like for other NetBox forms you can pass URL query parameters. I‘ve added a custom link to the device detail page with a link to the script. The query parameter should be in the form `varname=objectid`.

**Reply by llamafilm** [reply]
Thanks!  I didn't know about this feature, this should work nicely.

**Reply by llamafilm** [reply]
Do you know how to make this work for a MultiObjectVar?  I can't figure out the right query param.

**Reply by alehaa** [reply]
Try adding `varname=objectid` multiple times, e.g. `?tags=1&tags=2`.
